js/index.js

```diff
@@ -1,7 +1,364 @@
+const User_List = document.getElementById('user-list');
+let USER_COUNT =0;
+
+// 해당 서버에 데이터 가져와 처리
+function Response_Api(){
+  fetch('https://js-todo-list-9ca3a.df.r.appspot.com/api/users')
+      .then((response) => response.json())
+      .then((myJson)=> {
+        for(let i=0;i<myJson.length;i++)
+          Get_Users(myJson,i);
+        Click_User();
+        document.querySelector('.user-create-button').addEventListener('click',Make_User);
+        document.querySelector('.user-delete-button').addEventListener('click',Delete_User);
+      })
+};
+
+//해당 데이터 내용을 가진 name 추가
+function Get_Users(data,i){
+  const $user_list = document.getElementById('user-list');
+  const button = `<button class="ripple" id=${++USER_COUNT}>${data[i].name}</button>`
+  const appRender = () => {
+    $user_list.innerHTML += button;
+  }
+  appRender();
+}
+
+// 클릭시 생성 혹은 삭제 핸들러 실행
+function Make_User(){
+  document.querySelector('.user-create-button').addEventListener('click',onUserCreateHandler)
+}
+function Delete_User(){
+  document.querySelector('.user-delete-button').addEventListener('click',onUserDeleteHandler)
+}
+
+//추가한 이름을 브라우저 및 서버에 적용
 const onUserCreateHandler = () => {
   const userName = prompt("추가하고 싶은 이름을 입력해주세요.");
+  // User의 이름은 최소 2글자 이상체크
+  let User_Enough_Length = userName.length>1 ? true : false;
+  if(User_Enough_Length) {
```

Comment:
```suggestion
  if(userName.length>1) {
```
```User_Enough_Length``` 변수의 경우 조건문을 확인하는 것 외에는 사용되지 않고 있는데 별도의 변수를 선언하지 않고 조건문에 명시하는게 더 깔끔할거 같습니다!